### ISSUE TITLE
getType() incorrectly reports UPC when provided with EAN

### ISSUE THREAD

**Issue by johnhunt** [violuke/php-barcodes#2]
0700867967774 is an EAN and not a UPC. 700867967774 is UPC.

```
$barcode = new \violuke\Barcodes\BarcodeValidator("0700867967774");
‌violuke\Barcodes\BarcodeValidator::__set_state(array(
   'barcode' => '0700867967774',
   'type' => 'UPC',
   'gtin14' => '000000700867967774',
   'valid' => true,
))

‌‌$barcode->getType();
‌UPC

‌‌$barcode->getBarcode();
‌0700867967774
```

We're going to fix this ourselves now, will create PR shortly.

**Comment by violuke** [comment]
Thanks for the feedback, John. You are right. If you submit a PR, I'll get it merged in ASAP and a bug-fix release made so you can use it in your project more easily. Thanks.

**Comment by violuke** [comment]
This has been resolved now 1b89d5ec713005fefaa040af707406cb81bd0066. Tests added too.

**Comment by johnhunt** [comment]
Thanks, sorry I never got round to that PR!
